include/MemoryManager.h

```diff
@@ -109,17 +109,17 @@ struct MmAllocator
 	typedef T value_type;
 	template< class U > struct rebind { typedef MmAllocator<U> other; };
 
-	T* allocate(std::size_t n)
+	T* allocate( std::size_t n )
 	{
 		return reinterpret_cast<T*>( MemoryManager::alloc( sizeof(T) * n ) );
 	}
 
-	void deallocate(T* p, std::size_t)
+	void deallocate( T* p, std::size_t )
 	{
 		MemoryManager::free( p );
 	}
 
-	typedef std::vector<T, MmAllocator<T> > vector;
+	typedef std::vector<T, MmAllocator<T>> vector;
```

Comment:
looks like the space here was removed again, have an error while compiling.
```
error: ‘>>’ should be ‘> >’ within a nested template argument list
```

Reply:
Right, should have remembered from 73728d79da858b2ee07db5df4e1bd032bdd89911